addons/website_sale_stock/views/website_sale_stock.xml

```diff
@@ -1,5 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+  <template id="assets_frontend" inherit_id="website.assets_frontend" name="Shop">
+      <xpath expr="." position="inside">
+        <script type="text/javascript" src="/website_sale_stock/static/src/js/website_sale_stock.js"></script>
+      </xpath>
+  </template>
+  <template id="products_out_of_stock_warning" inherit_id="website_sale.product" active="False" customize_show="True" name="Out of Stock Warning">
```

Comment:
if we do it , then we cant render message on variant change eg. if product with color "white" have stock `0`  quantity, then we need to display message `Sorry ! This product is out of stock` , and now user change same product's variant color "black" have stock `5` quantity and user ask for `10` quantity then we have to display `Sorry ! Only 5 units are still in stock` and if user change variant "red" of same product have `100` quantity and end user ask for `10` then no message and he can proceed further.